Nasal/MCDU/INITA.nas

```diff
@@ -96,9 +96,9 @@ var initInputA = func(key, i) {
 			}
 		}
 	} else if (key == "R3") {
-		#if (getprop("/controls/adirs/mcducbtn") == 0) {
-		#	setprop("/controls/adirs/mcducbtn", 1);
-		#}
+		if (getprop("/controls/adirs/mcducbtn") == 0) {
+			setprop("/controls/adirs/mcducbtn", 1);
+		}
```

Comment:
It should be kept commented out. The "IRS Align" line can be selected, when you need to correct the automatically recorded position. This is not yet implemented.

Reply:
👍